.idea/runConfigurations/Local_Jetty_Server.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Local Jetty Server" type="Application" factoryName="Application">
-    <option name="ALTERNATIVE_JRE_PATH" value="1.8" />
+    <option name="ALTERNATIVE_JRE_PATH" value="liberica-17" />
```

Comment:
Maybe choose a more general name here...

Reply:
I have pushed a commit to address this.